### PR TITLE
Update new testnet spec

### PR DESCRIPTION
**Pull request by drewstone**
Gonna restart the testnet.

### CODE REVIEW

**Comment by raykyri** [comment]
Should add build instructions for the Beresheet testnet (`cargo build --release --features beresheet-runtime` is that right?)

**Comment by drewstone** [comment]
Yep! @raykyri forgot to push that change.

**Comment by raykyri** [comment]
Starting with the next release, we can also reset the Beresheet version number to mainnet, since the runtime name is different now.

**Comment by drewstone** [comment]
You can’t go backwards in runtime versions.

**Review by nblogist — APPROVED** [review]
LGTM!

File: node/cli/src/chain_spec.rs [hunk 1]
```diff
@@ -240,7 +233,7 @@ pub fn testnet_genesis(
 				.collect::<Vec<_>>(),
 		},
 		pallet_staking: StakingConfig {
-			validator_count: 20,
+			validator_count: 7,
```
[the remark this answers is not in the document]
Comment (drewstone) [reply node/cli/src/chain_spec.rs:236]: Because less is more.

File: node/runtime/src/lib.rs [hunk 2]
```diff
@@ -141,6 +142,21 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	transaction_version: 2,
 };
 
+#[cfg(feature = "beresheet-runtime")]
+pub const VERSION: RuntimeVersion = RuntimeVersion {
+	spec_name: create_runtime_str!("beresheet"),
+	impl_name: create_runtime_str!("beresheet-node"),
+	authoring_version: 16,
+	// Per convention: if the runtime behavior changes, increment spec_version
```
[the remark this answers is not in the document]
Comment (drewstone) [reply node/runtime/src/lib.rs:150]: The conventions are described there.

[the remark this answers is not in the document]
Comment (nblogist) [reply node/runtime/src/lib.rs:150]: Thank you

**Pull request merged**